src/style.css

```diff
@@ -1 +1,55 @@
 /* Styles go here */
+body {
+  margin: 0;
+}
+
+.container {
+  margin: 0;
+  display: flex;
+}
+
+.logo {
+  margin-top: 10px;
+  margin-left: 50px;
+}
+
+.navigation {
+  width: 100vw;
+  font-family: "Roboto", sans-serif;
+  text-transform: uppercase;
+  font-size: 12px;
+}
+
+.list__element {
+  height: 60px;
+  line-height: 60px;
+  margin: 0 20px 0 0;
+  list-style-type: none;
+  cursor: pointer;
+}
+
+.nav__link {
+  text-decoration: none;
+  color: black;
+  padding: 20px 0;
```

Comment:
With line-height I think you dont need to set padding for nav__link, remove this if it is unnecessary